.gitignore

```diff
@@ -80,3 +80,35 @@ ircmaxell/random-lib/composer.lock
 ircmaxell/random-lib/phpunit.xml.dist
 
 icewind/streams/tests
+
+guzzlehttp/guzzle/build/
+guzzlehttp/guzzle/docs/
+guzzlehttp/guzzle/tests/
+guzzlehttp/guzzle/.gitignore
+guzzlehttp/guzzle/.travis.yml
+guzzlehttp/guzzle/CHANGELOG.md
+guzzlehttp/guzzle/composer.json
+guzzlehttp/guzzle/LICENSE
+guzzlehttp/guzzle/MAKEFILE
+guzzlehttp/guzzle/phpunit.xml.dist
+guzzlehttp/guzzle/README.md
+guzzlehttp/guzzle/UPGRADING.md
+guzzlehttp/ringphp/docs/
+guzzlehttp/ringphp/tests/
+guzzlehttp/ringphp/.gitignore
+guzzlehttp/ringphp/.travis.yml
+guzzlehttp/ringphp/CHANGELOG.md
+guzzlehttp/ringphp/LICENSE
+guzzlehttp/ringphp/Makefile
+guzzlehttp/ringphp/README.rst
+guzzlehttp/ringphp/composer.json
+guzzlehttp/ringphp/phpunit.xml.dist
+guzzlehttp/streams/tests/
+guzzlehttp/streams/.gitignore
+guzzlehttp/streams/.travis.yml
+guzzlehttp/streams/CHANGELOG.rst
+guzzlehttp/streams/LICENSE
+guzzlehttp/streams/Makefile
+guzzlehttp/streams/README.rst
+guzzlehttp/streams/composer.json
+guzzlehttp/streams/phpunit.xml.dist
```

Comment:
maybe add following general rules:

```
.travis.yml
README*
tests
docs
Makefile
phpunit.xml.dist
```

This should shrink this a lot and makes it a lot more maintainable ;)